packages/opentelemetry-tracing/src/Span.ts

```diff
@@ -51,7 +51,7 @@ export class Span implements api.Span, ReadableSpan {
   readonly instrumentationLibrary: InstrumentationLibrary;
   name: string;
   status: api.Status = {
-    code: api.CanonicalCode.OK,
+    code: api.StatusCode.UNSET,
```

Comment:
Why do we have "UNSET" now as default ?
Is this required by spec if so where is it? 
Currently all spans are being exported with status "UNSET" previously if they were ok they were exported as "OK"
@dyladan ^^

Reply:
OK I have found this to be set as default, but shouldn't the span be set to "OK" once it is finished and there were no errors ?